src/psyclone/tests/psyir/transformations/intrinsics/matmul2code_trans_test.py

```diff
@@ -568,12 +571,13 @@ def test_validate14():
                                       Literal("15", INTEGER_TYPE))
     with pytest.raises(TransformationError) as excinfo:
         trans.validate(matmul)
-    assert ("Transformation Error: To use matmul2code_trans on matmul, "
-            "no more than two indices of the argument 'x' "
-            "must be full ranges but found 3." in str(excinfo.value))
+    assert (f"Transformation Error: To use {trans.name} on matmul, "
```

Comment:
No f-string...

Reply:
Addressed in c4d60bc3ef52a6d7b2853dd5b3c3b01ce82996dc.